### DISCUSSION TITLE
Could using Gluetun impede a Sonarr container from seeing a downloader container running through Glutun?

### DISCUSSION THREAD

**Discussion by aprillia99** [qdm12/gluetun#372] [Ideas]
Have a transmission container running through Gluetun. Sonarr can't seem to find/access it from the localhost and port I use to access the web dashboard. Could gluetun create a problem like this? or is it something else in my config?

**Comment by qdm12** [comment]
It shouldn't but maybe share your configuration (and remove credentials)? It depends, where is sonarr running? In the same docker network or through gluetun as well?

**Reply by aprillia99** [reply]
Here is the basic config

https://pastebin.com/AcxiE3L0

**Reply by qdm12** [reply]
Alright, you need gluetun and sonarr to be on the same Docker network. For example you can set `network_mode: bridge` on both docker-compose.yml. Then sonarr should be able to reach `gluetun:9999` where `9999` is your transmission port.

**Reply by qdm12** [reply]
Hey there! Did it work in the end? Thanks
